src/components/workflow/workflow.js

```diff
@@ -241,15 +239,19 @@ export class Workflow extends Component {
 
     // Apply animating class to zoom wrapper
     select(this.wrapperRef.current).classed(
-      'pipeline-workflow-wrapper--animating',
+      '.pipeline-workflow__zoom-wrapper--animating',
       true
     );
-
     // Update layer label y positions
     if (this.layerNamesRef?.current) {
       const layerNames = this.layerNamesRef.current.querySelectorAll(
         '.pipeline-layer-name'
       );
+      if (layerNames.length !== this.props.layers.length) {
```

Comment:
do we know why is this happening in the workflow view?

Reply:
hey im not sure yet but i created a ticket to address the wider issue about workflow and flowchart here https://github.com/kedro-org/kedro-viz/issues/2386